packages/edit-post/src/components/visual-editor/index.js

```diff
@@ -158,27 +162,21 @@ export default function VisualEditor( { styles } ) {
 			wrapperBlockName: _wrapperBlockName,
 			wrapperUniqueId: getCurrentPostId(),
 			isBlockBasedTheme: editorSettings.__unstableIsBlockBasedTheme,
+			// WARNING: use getEditorSettings from the editor store, not the
+			// block editor store. The settings on the block editor store set
+			// with a delay and we need the assets on the first render for the
+			// iframe srcDoc.
```

Comment:
I think that's not the right question. We shouldn't be switching stores to use because the settings are not set yet.

IMO, we should ask ourselves whether these settings are supposed to be generic "block editor" settings, if yes, we should continue to use the block editor store and move the useSelect to inside the "iframe" component. If not, we should remove them from the block editor settings entirely.

Reply:
That's a good point. It shouldn't really be in block editor settings at all I think.

Reply:
Seems like we do need it for the BlockPreview component, which is in the block-editor package: https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/components/block-preview/auto.js#L40.

Reply:
Can we then restore the block editor selectors and move them inside the iframe component instead, that way they are included by default in all iframes?  I guess this also means we need an "isReady" somewhere, I don't really see a way around it personally?

Reply:
Done. How does it look to you now?